### ISSUE TITLE
Thunderhub does not create a "fresh/new" onchain address

### ISSUE THREAD

**Issue by thespielplatz** [apotdevin/thunderhub#534]
**Describe the problem/bug**
When creating a new onchain address via cli, this "new address" is also shown in thunderhub. 

I don't know if this is really a bug, but in my case it lead to an account problem with another application, while rebalancing BTC via Boltz exchange.

**Your environment**
* Version of ThunderHub: 0.13.19
* Deployment method: docker

**To Reproduce**
1. Create new onchain address via lncli
![Screenshot 2023-05-31 at 11 23 53](https://github.com/apotdevin/thunderhub/assets/3327667/137429ef-09d5-4d56-89da-34d26abe4598)

2. Create new onchain address via Thunderhub
![Screenshot 2023-05-31 at 11 28 25](https://github.com/apotdevin/thunderhub/assets/3327667/3132e81c-4499-44a3-81ce-19c62dc56584)

**Expected behavior**
Thunderhub creates a new onchain address

**Actual behavior**
Thunderhub shows the previous created address

**Comment by tompro** [comment]
Yes we ran into a similar issue (LND). From what I gathered in the newAddress RPC and rest the address type needs to be one of the NOT unused ones: 
![image](https://github.com/user-attachments/assets/96ce3d4a-8ef4-40b2-a7c9-235fa5bc300c)

A type prefixed UNUSED_ will result in one of the addresses that have already been generated but have not been used. With the other types it should always generate a fresh one. https://github.com/lightningnetwork/lnd/issues/8697
